control_plane/src/endpoint.rs

```diff
@@ -750,6 +750,7 @@ impl Endpoint {
                 endpoint_storage_addr: Some(endpoint_storage_addr),
                 endpoint_storage_token: Some(endpoint_storage_token),
                 autoprewarm: false,
+                suspend_timeout_seconds: 3600, // TODO(thesuhas): Setting it to 1 hour for now, figure out how to pass it.
```

Comment:
This should be 0, right? We don't autosuspend neon_local computes

Reply:
Maybe ask a control plane person what the default for a compute that never suspends is. I think it is -1, but 🤷

Reply:
We should also make the default on the CLI struct the unlimited value.